Makefile

```diff
@@ -35,10 +35,10 @@ coverage: clean ## generate and view HTML coverage report
 export CUSTOM_COMPILE_COMMAND = make upgrade
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -q pip-tools
-	pip-compile --rebuild --upgrade -o requirements/dev.txt requirements/base.in requirements/dev.in requirements/quality.in requirements/test.in requirements/travis.in
-	pip-compile --rebuild --upgrade -o requirements/quality.txt requirements/base.in requirements/quality.in requirements/test.in
-	pip-compile --rebuild --upgrade -o requirements/test.txt requirements/base.in requirements/test.in
-	pip-compile --rebuild --upgrade -o requirements/travis.txt requirements/travis.in
+	pip-compile --rebuild -o requirements/dev.txt requirements/base.in requirements/dev.in requirements/quality.in requirements/test.in requirements/travis.in
```

Comment:
Accidental?

Reply:
yes... :tired_face:

Reply:
I make this mistake so frequently, I wonder how other Open edX committers avoid it...

Reply:
We have tooling to automate changes across repos. I wonder if I sent you docs, if you’d want to try to make the change you made in edx-platform across repos. This wouldn’t be blended work, but if it would make you happy.